corehq/warehouse/tests/test_validate.py

```diff
@@ -3,6 +3,30 @@
 
 
 def test_validate_loaders():
-    for slug in ALL_TABLES:
+    def check(slug):
         loader = get_loader_by_slug(slug)
         loader().validate()
+
+    for slug in ALL_TABLES:
+        yield check, slug
+
+
+def check_for_loader_cycles(slug):
```

Comment:
in what scenario will this test reveal an issue?

Reply:
e.g. app_status_fact -> app_status_form_staging -> app_status_fact

It's not a real issue since these dependencies aren't really used for anything in code but it does help with understanding the flow without having to go and check airflow.